var/spack/repos/builtin/packages/seacas/package.py

```diff
@@ -179,6 +183,15 @@ class Seacas(CMakePackage):
         when="@:2021-01-20",
         msg="The Faodel TPL is only compatible with @2021-04-05 and later.",
     )
+    conflicts("+shared", when="platform=windows")
+    conflicts("+fortran", when="platform=windows")
```

Comment:
Did you mean `+x11`? I didn't recall an issue with fortran in the original edits.

Reply:
Whoops, yeah, I had exodus on the brain (doesn't help they vendor each other), thanks good catch

Reply:
Fixed and pushed.